tests/orquestra/integrations/cirq/simulator/qsimsimulator_test.py

```diff
@@ -0,0 +1,161 @@
+################################################################################
+# © Copyright 2021-2022 Zapata Computing Inc.
+################################################################################
+import numpy as np
+import pytest
+from cirq import depolarize
+
+# from cirq import depolarize
+from orquestra.quantum.api.backend_test import (
+    QuantumSimulatorGatesTest,
+    QuantumSimulatorTests,
+)
+from orquestra.quantum.circuits import CNOT, Circuit, H, X
+from orquestra.quantum.openfermion.ops import QubitOperator
+
+from orquestra.integrations.cirq.simulator.qsim_simulator import QSimSimulator
+
+
+@pytest.fixture()
+def backend():
+    return QSimSimulator()
+
+
+@pytest.fixture()
+def wf_simulator():
+    return QSimSimulator()
+
+
+@pytest.fixture()
+def sampling_simulator():
```

Comment:
Why do we need to define this?

Reply:
I was part of the original code. I can remove this in both cirq and qsim simulator.